buildtools/dep/dep.go

```diff
@@ -13,6 +15,25 @@ import (
 
 // ErrNoLockfile is returned if a dep manifest is found without an accompanying lockfile.
 var ErrNoLockfile = errors.New("found Gopkg.toml but not Gopkg.lock")
+var ErrIsIgnored = errors.New("this is ignored")
```

Comment:
Is this ever used?

Reply:
No, that was actually left over from our remote session. Good catch I never changed that line but should have.